portability-core/src/main/java/org/dataportabilityproject/serviceProviders/google/contacts/VCardToGoogleContactConverter.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2018 The Data-Portability Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dataportabilityproject.serviceProviders.google.contacts;
+
+import com.google.api.services.people.v1.model.EmailAddress;
+import com.google.api.services.people.v1.model.FieldMetadata;
+import com.google.api.services.people.v1.model.Name;
+import com.google.api.services.people.v1.model.Person;
+import com.google.api.services.people.v1.model.PhoneNumber;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Preconditions;
+import ezvcard.VCard;
+import ezvcard.property.Email;
+import ezvcard.property.StructuredName;
+import ezvcard.property.Telephone;
+import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class VCardToGoogleContactConverter {
```

Comment:
Another idea, depending what you think is to expose this converter and the reverse by extending an actual Converter type and delegate to these specific implementations? 

https://google.github.io/guava/releases/17.0/api/docs/com/google/common/base/Converter.html

Reply:
Let's talk with Jim about this idea.